listenbrainz/webserver/templates/index/index.html

```diff
@@ -74,6 +74,21 @@ <h2 class="page-title">Welcome to ListenBrainz!</h2>
             {% endif %}
             </tbody>
           </table>
+          <table class="table table-collapsed">
+            <thead>
+              <tr>
+                <th>Latest Listens</th>
+              </tr>
+            </thead>
+            <tbody>
+            {% if latest_listens %}
+              <tr>
+                <td>Latest listens:
+                <td>{{ "{:,}".format(latest_listens) }}</td>
```

Comment:
We should only show the track name and the artist name here, in my opinion.

Reply:
Still unsure of how can this be implemented.. Maybe add some codes in `index.py`?

Reply:
yes, just pass alist of dict in the format 

```
{
'artist_name': 'Kanye West',
'track_name': 'Fade',
}
```

from index.py into the jinja template and show listen.artist_name and listen.track_name in index.html

Reply:
can I write it in the `get_latest_listens` function?
or shoudl I add this in `index.py`?